innersource-guidance-group.md

```diff
@@ -0,0 +1,43 @@
+
+## Problem
+Without a set of organizational standards Trusted Committers, and individual teams, are free to set their own standards, which is a step in the right direction.  However, team standards could be highly variant and further diverge over time as they evolve and adapt to new needs and forces.  This can lead to confusion and frustration as contributors submit code across team and functional boundaries. The business is interested establishing a set of agreed-upon standard operating procedures that guide teams while still providing sufficient autonomy to meet the parochial needs of individual teams.
+
+## Context
+A company is planning to introduce InnerSource as a strategic initiative. They are faced with reforming and steering the behavior of developers and engineers toward a common goal.  In general functional groups and teams within the company adhere to a broad spectrum of standards regarding tools, development languages, collaboration models, engagement models, documentation models, etc. The company recognizes the value of the Trusted Committer role as a situational leader for a team that sets and enforces team standards, code contribution standards, contribution agreements, PR management, etc. however the company is interested in establishing organizational guidelines as well that steer and focus Trusted Committers and the teams they represent.
+
+As an example, the breadth of languages used within an engineering and operations organization could include, powershell, bash shell, perl, tcl, python, golang, java, shell augmented with sed and awk and jq, etc. and the organization wants to standardize as much as possible on a subset of these languages so that the ability to collaborate between teams is maximized.
+
+## Forces
+- Separate engineering and development teams establish development standards that may be highly divergent from other teams which limits the ability to collaborate.
+- As people move between teams, or responsibility for specific initiatives change over time, the ability of the organization to adapt to variant team norms results in decreased development velocity.
+
+## Solutions
+Recognize that Trusted Committers are both situational leaders and subject matter experts representing their team and establish a governance model with a “committee” attended and chaired by Trusted Committers chartered with establishing company and/or organizational guidelines.
+
+- The “committee” is scheduled periodically as necessary by the TC’s – twice a week, once a month, quarterly, etc.
+- The goal is to establish general direction, or a highway, while leaving freedom for individual teams to adapt to local needs, or side streets
+- Trusted Committers establish organization guidelines that become templates for Contribution Agreements
+- Trusted Committers collaborate on and refine guidelines asynchronously via passive documentation and the establishmentment of organizational patterns
```

Comment:
I, personally, see this as an inner source project, whose contributors and community are those at the company who are interested in inner source.

This project should live and run itself by its own advice that it gives to others.  It just happens to be a collaborative documentation project as opposed to a collaborative code project.

Reply:
+1 to what @rrrutledge said. This is also something that has worked very well for us at Europace: Our very first InnerSource project was one tasked with bringing people interested in InnerSource together, documenting best practices along the way and helping out with advice for others.

Reply:
Thanks for jumping in to comment, @MaineC ❗️

Reply:
Reworded this as such:

> - Trusted Committers collaborate on and refine guidelines asynchronously via passive documentation and the establishment of organizational patterns. 
>     - This guideline definition project itself can be run as an InnerSource project, accepting contributions from anywhere in the organization. 
>     - By running this as a collaborative documentation project, it can role-model and verify the behaviors that they want to promote for other InnerSource projects as well.